content/en/docs/tasks/virtual-machines/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Virtual Machines
```

Comment:
This doesn't belong as top level heading in Tasks. The 4 current Task categories align with the Istio's feature categories (Traffic Management, Security, Policies, Observability). Virtual Machines is not another feature category.

These aren't really tasks either, they're setup (deployment) patterns. These should either be moved to the Setup section or leave them in Examples, where they currently are.

Reply:
The docs mention "Installation steps" but they are more about adding workloads running on VMs to existing single network or a multi-network deployments and how to setup the mesh to do so. 
Therefore, the content is more operational and how-to-focused than an example. That's why I placed it in "Tasks".
@frankbu I could move it to "Operations" but "Setup" would also not be the right place IMO. WDYT?